src/components/atoms/FileButton/FileButton.tsx

```diff
@@ -45,6 +45,7 @@ export const FileButton: React.FC<FileButtonProps> = ({
         hidden
         type="file"
         id="fileButton"
+        accept=".png, .jpg, .jpeg"
```

Comment:
Maybe import and use `ACCEPTED_IMAGE_TYPES` ?

Reply:
fixed in 24f391d2ee6cf6ad1d3364c161e287325e75ab44